x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -218,7 +218,8 @@ export const getManagementFilteredLinks = async (
     fleetAuthz && currentUser
       ? calculateEndpointAuthz(licenseService, fleetAuthz, currentUser.roles)
       : getEndpointAuthzInitialState();
-
+  const isPlatinumOrTrialLicense =
+    licenseService.isPlatinumPlus() || licenseService.getLicenseInformation()?.type === 'trial';
```

Comment:
Is there any specific reason to check for TRIAL? Isn't trial above Platinum? ;)

Reply:
Agreed, addressed in 1bc331426d153b39beebec5b15063082795e2247.